### ISSUE TITLE
question about deeplab_resnet_init.ckpt and deeplab_resnet.ckpt

### ISSUE THREAD

**Issue by gaopeng-eugene** [DrSleep/tensorflow-deeplab-resnet#19]
deeplab_resnet_init.ckpt is the original caffe resnet trained on IMAGENTE by Kai Ming He while deeplab_resnet.ckpt is fintuned on MS-COCO after adding atrous CNN. 
Do you know how to finetune on the MS-COCO dataset? They use classification or segmentation to finetune?

**Comment by gaopeng-eugene** [comment]
deeplab_resnet.ckpt is transformed from init.caffemodel or not?

**Comment by DrSleep** [comment]
[Here](https://drive.google.com/open?id=0B_rootXHuswsZ0E4Mjh1ZU5xZVU):
init.caffemodel -> deeplab_resnet_init.ckpt
train2_iter20000.caffemodel -> deeplab_resnet.ckpt

As I know, everyone uses MS-COCO segmentation. For example, the CRFasRNN paper contains particular details on how they exploited COCO.

**Comment by gaopeng-eugene** [comment]
Do you mean init.caffemodel is the original residual model while train2_iter20000.caffemodel is fine tuned on the MS_COCO model?

**Comment by DrSleep** [comment]
No, train2_iter20000 is init fine-tuned on PASCAL.

From [this page](http://liangchiehchen.com/projects/DeepLabv2_resnet.html): 

> Note that the provided init.caffemodel has already been pretrained on MS-COCO.

**Comment by siinem** [comment]
Dear @DrSleep 

Sorry, i still couldnot understand this clearly.. Do you mean (at your last message) that train2_iter20000 is initialized by model learned on MS COCO dataset and then fine-tuned by PASCAL VOC images (so just the weights of last layer is re-learned-?-)? 

When i use deeplab_resnet_init.ckpt on fine_tune.py to apply the technique on my dataset (so pretrained by MS COCO-origined model and weights of last layer are tuned for my own data) i get better performance result on the validation set, compared to the same procedure implemented by employing deeplab_resnet.ckpt .  I was expecting that performances of these would not differ a lot because both use by ms-coco pretrained model.  Do you think this is reasonable?
